Test/Unit/Helpers/PaymentPlansHelperTest.php

```diff
@@ -112,6 +112,50 @@ public function testFormatFeePlanConfigForBackOfficeDisplay($apiFeePlan, $config
         $this->assertEquals($result, $this->createPaymentPlansHelper()->formatLocalFeePlanConfig($apiFeePlan, $configFeePlan));
     }
 
+    /**
+     * @return void
+     */
+    public function testIsInPageAllowedWithInPageActivated()
+    {
+        $planKeys = $this->getInPagePlanKeys();
```

Comment:
You can create a dataProvider for testing separatly each plan

Reply:
@Francois-Gomis Done

Reply:
assertFalse c'est nickel je l'avais pas vu